### PR TITLE
Create voidtools.com.xml

### PR DESCRIPTION
**Pull request by renyouguo**

### CODE REVIEW

**Comment by J0WI** [comment]
Could you also redirect `http://www.forum.voidtools.com/` to `https://www.voidtools.com/forum/`?
(You should double check if they are really the same.)

**Comment by J0WI** [comment]
Of course you should comment that this is necessary because of the mismatch on `www.forum.voidtools.com`. Sorry that this was not clear.

**Comment by renyouguo** [comment]
@J0WI What should I do? I cann't understand the check error.

File: src/chrome/content/rules/voidtools.com.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<ruleset name="voidtools.com">
+
+	<!--	Mismatch on  www.forum.voidtools.com	-->
+
+	<target host="www.forum.voidtools.com"/>
+	<rule from="^http://www\.forum\.voidtools\.com/" to="https://www.voidtools.com/forum/" />
+		<test url="http://www.forum.voidtools.com/" />
+		
+`	<!--	Directly	-->
```
Comment (J0WI) [inline src/chrome/content/rules/voidtools.com.xml]: Remove the grave accent here on col 1 to fix syntax errors.

**Pull request merged**